source/_components/switch.telnet.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Telnet Switch"
+description: "Instructions how to integrate telnet switches into Home Assistant."
+date: 2017-08-10 19:19
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Switch
+ha_release: 0.51
+ha_iot_class: "Local Polling"
+---
+
+
+The `telnet` switch platform allows you to control devices with telnet commands. It connects to a host on port 23.
```

Comment:
Telnet is typically used over port 23 but there's some applications and hardware that may use different ports.